### PR TITLE
accounts: add label and AccountInfo RPC

### PR DESCRIPTION
**Pull request by guggero**
Fixes https://github.com/lightninglabs/lightning-terminal/issues/592.
Fixes https://github.com/lightninglabs/lightning-terminal/issues/581.

Adds a label to custodial accounts and allows individual accounts to be queried through the RPC or CLI (instead of needing to list all accounts).

### CODE REVIEW

File: cmd/litcli/accounts.go [hunk 1]
```diff
@@ -293,29 +348,59 @@ func removeAccount(ctx *cli.Context) error {
...
-	}
-	if _, err := hex.DecodeString(accountID); err != nil {
-		return err
-	}
+		// Since we have a positional argument, we cannot be sure it's
+		// an ID. So we check if it's an ID by trying to hex decode it
+		// and by checking the length. This will break if the user
+		// chooses labels that are also valid hex encoded IDs. But since
+		// the label is supposed to be human-readable, this should be
+		// unlikely.
```
Comment (ViktorTigerstrom) [inline cmd/litcli/accounts.go:392]: nit: It shouldn't be possible for the user to set such labels, no?

**Review by ViktorTigerstrom — APPROVED** [review]
LGTM! Great additional features to add 🚀!

Just leaving one tiny nit that's non blocking.

File: cmd/litcli/accounts.go (hunk 1 above)
Comment (guggero) [reply cmd/litcli/accounts.go:392]: Good point, I added a check here: https://github.com/lightninglabs/lightning-terminal/blob/568d7ad508933df0bcb4603b48594d3d0558f57b/accounts/store.go#L119

Comment (ViktorTigerstrom) [reply cmd/litcli/accounts.go:392]: Ah yeah, awesome! I realize that my comment above was a bit unclear, but I ment to say that the part of this comment I've marked isn't necessary, and can be removed as it shouldn't be possible for users to set such labels :). Though feel free to ignore this, as this isn't blocking at all and very nit picky!

**Review by bitromortac — APPROVED** [review]
LGTM :zap:, great work!

**Pull request merged**